containers/conformance.md

```diff
@@ -0,0 +1,220 @@
+# Zowe Conformance Criteria Related to Containerization
+
+**DRAFT**
+
+These conformance criteria are applicable for all Zowe components intent to run in containerization environment. The containerization environment could be Kubernetes or OpenShift running on Linux or Linux on Z. This may also apply to `docker-compose` running on Linux, Windows, Mac OS, or zCX.
+
+## Image
+
+In general, the image should follow [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/). Below requirements are in addition to the list.
+
+### Base Image
+
+You are free to choose base image based on your requirements.
+
+For Zowe core components, here are our recommendations:
+
+- [Red Hat Universal Base Image 8 Minimal](https://developers.redhat.com/articles/ubi-faq?redirect_fragment=resources#ubi_details)
+- [Ubuntu](https://hub.docker.com/_/ubuntu)
+
+The image should be as small as possible.
+
+### Multi-CPU Architecture
+
+- Zowe core components must release images based on both `amd64` and `s390x` CPU architecture.
+- Zowe core component images must use multiple manifests to define the image supports multiple CPU architecture.
+- CPU architecture like `amd64` and `s390x` must NOT appear in released component image tag.
```

Comment:
why not? it helps in certain cases and its not a problem with aliases
latest =  latest-ubuntu = 1.23-ubuntu = 1.23-ubuntu-amd64 = latest-amd64
i think the real *MUST* is components must use manifest when in a registry, so that you can alias latest-amd64 and latest-s390x to just "latest".

Reply:
This is for formal release image. For dev/test purpose, I guess it's ok.